Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Hilmar Wiegand <me@hwgnd.de>", "Amethyst Foundation <contact@amethys
 edition = "2018"
 
 [dependencies]
-amethyst = {version = "0.15.2", features = ["no-slow-safety-checks"]}
+amethyst = { path = "../amethyst", features = ["optional"] }
```

Comment:
I don't think we want to rely on the user having cloned amethyst although I wouldn't mind more contributions ...

Reply:
I couldn't get it to compile on 0.15.3. But that's to be expected, right? The latest version in crates.io hasn't been migrated to legion yet.

One option would be to hold off on merging this until 0.16 is out. 

For my own purposes, I wanted to start development on a new game by using the legion version of amethyst, rather than be forced into a migration later on once 0.16 comes out. That's why I migrated the starter project. Anyone else in the same boat could use my fork for the time being.

Reply:
Wouldn't it make more sense, for now at least, to explicitly point to a rev or a tag of the amethyst repo? At least that will make it clear that the change is not in prod yet, but still let it compile

Reply:
@rdelfin That's a good idea. I'd rather not point to a local clone. I'm not sure how to do what you're suggesting, though. Any chance you could point me to some documentation?

Reply:
@nyghtly-derek sure! You can specify the `git` attribute instead and point it to the http url of the repo, as well as a `rev` attribute, which is just the git hash. More here: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories

Reply:
@rdelfin Done! Thank you for your help 😄